### PR TITLE
Check 'Status' to determine VM state & add retry

### PR DESCRIPTION
**Pull request by santoshx**
We dont need just start-vm command to succeed.
We need VMs to be in running state.

Without this my tests with three or more VMs fails at least once in three runs.

### CODE REVIEW

File: Libraries/HyperV.psm1 [hunk 1]
```diff
@@ -544,16 +544,30 @@ Function StartHyperVGroupVMs($HyperVGroupName,$HyperVHost)
     $CurrentErrors = @()
     foreach ( $VM in $AllVMs.VMMembers)
     {
+        $RetryCount = 5
+        $CurrentRetryAttempt=0
+        $res = "False"
```
Comment (lizzha) [inline Libraries/HyperV.psm1]: use $false, $true

File: Libraries/HyperV.psm1 [hunk 2]
```diff
@@ -544,16 +544,30 @@ Function StartHyperVGroupVMs($HyperVGroupName,$HyperVHost)
     $CurrentErrors = @()
     foreach ( $VM in $AllVMs.VMMembers)
     {
+        $RetryCount = 5
```
Comment (ader1990) [inline Libraries/HyperV.psm1]: why not use Wait-VMState method? it is more reliable

File: Libraries/HyperV.psm1 (hunk 1 above)
Comment (ader1990) [reply Libraries/HyperV.psm1]: also, what you are saying is that your vms do not start correctly? what is the reason for that?

usually vms don t start when they don t have enough resources like ram/cpu or badly configured hardware.

Comment (santoshx) [reply Libraries/HyperV.psm1]: After this your comment I observed failures for a day and found all failures were due to lack of resources. This PR is not needed. Thank you.

**Pull request closed without merging**